paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -1247,46 +1293,40 @@ def get_splunk_api_token(self) -> SplunkAuth:
         )
 
     def get_button_text(self, button: str, is_active: bool) -> str:
-        # Button text max length 75 characters
-        # Current button templates allow version max length of 36
-        version_short_str = self.deployment_version.short_sha_repr()
-        if len(version_short_str) > 36:
-            # we'll have to depend on subsequent slack messages to show full version
-            version_short_str = "new version"
+        # TODO: Update button text for image_version
+        #  Will require thought on shortening
         active_button_texts = {
-            "forward": f"Rolling Forward to {version_short_str} :zombocom:"
+            "forward": f"Rolling Forward to {self.commit[:8]} :zombocom:"
```

Comment:
we should probably stick with an approach like the previous one where we refer to a single variable that's formatted as we want rather than do the string manipulation everywhere that we need this value